basis/models.py

```diff
@@ -35,9 +35,12 @@ class PersistentModel(models.Model):
     class Meta:
         abstract = True
 
-    def delete(self, *args, **kwargs):
-        self.deleted = True
-        self.save(*args, **kwargs)
+    def delete(self, using=None, force=False):
```

Comment:
Why did you introduce using? 

Can you please write a test when using is used?